### PR TITLE
Fix typo in nudb tool

### PR DESCRIPTION
**Pull request by MarkusTeufelberger**
"]" --> ">"

### CODE REVIEW

File: tools/nudb.cpp [hunk 1]
```diff
@@ -166,7 +166,7 @@ class admin_tool
             "        file is present.  Running commands on an unrecovered database\n"
             "        may result in lost or corrupted data.\n"
             "\n"
-            "    rekey <dat-path] <key-path> <log-path> --count=<items> --buffer=<bytes>\n"
+            "    rekey <dat-path> <key-path> <log-path> --count=<items> --buffer=<bytes>\n"
```
Comment (vinniefalco) [inline tools/nudb.cpp]: \facepalm

**Comment by MarkusTeufelberger** [comment]
You need anything else? This is now half a year in limbo...

**Comment by vinniefalco** [comment]
Sorry, I've been really focused on getting Beast ready for the Boost formal review. When I work on another project it requires switching gears and tuning my development setup for that project, and I am not ready to make that context switch just yet.

**Comment by MarkusTeufelberger** [comment]
No problem, I was just wondering since I'm writing Python bindings for this, if the project is abandoned or not accepting pull requests.

**Comment by vinniefalco** [comment]
Definitely not abandoned!

**Comment by MarkusTeufelberger** [comment]
Apparently this commit got cherry-picked and merged by Vinnie some time in 2017, I'm closing this PR.

**Pull request closed without merging**